src/routes/index.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Home from "../components";
+import Login from "../components/login";
+import Nav from "../components/nav";
+import {BrowserRouter,Route,Switch} from "react-router-dom";
+import NotFound from "../components/error";
+
+export default function AppRoutes(){
+	return(
+		<BrowserRouter>
+			<div>
+				<Nav/>
+				<Switch>
+					<Route path="/" component={Home} exact strict/>
+					<Route path="/login" component={Login} exact strict/>
+					<Route component={NotFound} e/>
+					{/* Add your routes here */}
+				</Switch>
+			</div>
+		</BrowserRouter>
+	);
+
```

Comment:
Could you explain why you created another component to hold the routes rather than have them directly held in App.js?

Reply:
@LehruAngela I did this because I did not want us to populate the App.js with all the routes. I think having a separate file for all the routes will make the code clean and more organized. Kindly correct me if am wrong.

Reply:
@LehruAngela should I go with what I have or I add all the routes to App.js?

Reply:
Yes, you can go with what you have already done. It is alright.